odxtools/diaglayers/diaglayerraw.py

```diff
@@ -149,6 +150,10 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) ->
             for el in et_element.iterfind("ADDITIONAL-AUDIENCES/ADDITIONAL-AUDIENCE")
         ]
 
+        librarys = [
```

Comment:
I don't think following the standard mistakes in naming is a good idea

Reply:
agreed. fixed: 5d7cf47d91ff2fce8867cca77aef7f94eb905084